instagram-app/src/components/PostContainer/User.jsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+import './Posts.scss'
+
+export default class User extends Component {
+    render() {
+        return (
+            <section className="d-flex user-head align-items-center">
+                {this.props.thumb &&
+                    <img src={this.props.thumb} alt={`Thumbnail ${this.props.thumb}`} className="icon"/>
+                }
+                <p className="p-1 mt-2">

```

Comment:
Nice use of the logical short-circuit with `&&` to rather than be added as a second conditional just calling the action you would like to run.